v2.0/show-create-sequence.md

```diff
@@ -0,0 +1,71 @@
+---
+title: SHOW CREATE SEQUENCE
+summary: The SHOW CREATE SEQUENCE statement shows the CREATE SEQUENCE statement that would create a carbon copy of the specified sequence.
+toc: false
+---
+
+The `SHOW CREATE SEQUENCE` [statement](sql-statements.html) shows the `CREATE SEQUENCE` statement that would create a carbon copy of the specified table.
+
+<div id="toc"></div>
+
+## Required Privileges
+
+The user must have any [privilege](privileges.html) on the target sequence.
```

Comment:
"any" sounds ambiguous. If any privileges are allowed, can we remove the Required Privileges section?

Reply:
Agreed - this is copied from the other `SHOW CREATE` docs. Instead of removing the section, maybe we should change all of them to say "No required privileges"?

Reply:
Thanks for the overview, @jseldess